esp-lp-hal/README.md

```diff
@@ -2,23 +2,18 @@
 
 [![Crates.io](https://img.shields.io/crates/v/esp-lp-hal?labelColor=1C2C2E&color=C96329&logo=Rust&style=flat-square)](https://crates.io/crates/esp-lp-hal)
 [![docs.rs](https://img.shields.io/docsrs/esp-lp-hal?labelColor=1C2C2E&color=C96329&logo=rust&style=flat-square)](https://docs.rs/esp-lp-hal)
+![MSRV](https://img.shields.io/badge/MSRV-1.76-blue?labelColor=1C2C2E&style=flat-square)
 ![Crates.io](https://img.shields.io/crates/l/esp-lp-hal?labelColor=1C2C2E&style=flat-square)
 [![Matrix](https://img.shields.io/matrix/esp-rs:matrix.org?label=join%20matrix&labelColor=1C2C2E&color=BEC5C9&logo=matrix&style=flat-square)](https://matrix.to/#/#esp-rs:matrix.org)
 
-`no_std` HAL for the low-power RISC-V coprocessors found on the ESP32-C6, ESP32-S2, and ESP32-S3 from Espressif.
+Bare-metal (`no_std`) hardware abstraction layer for the low-power RISC-V coprocessors found in the ESP32-C6, ESP32-S2, and ESP32-S3 from Espressif.
 
-Implements a number of the traits defined in [embedded-hal](https://github.com/rust-embedded/embedded-hal).
+Implements a number of blocking and, where applicable, async traits from the various packages in the [embedded-hal] repository.
 
-These devices uses the RISC-V ISA, which is officially supported by the Rust compiler via the `riscv32imc-unknown-none-elf` and `riscv32imac-unknown-none-elf` targets.
+For help getting started with this HAL, please refer to [The Rust on ESP Book] and the [documentation].
```

Comment:
Since it has not been published yet, this documentation link and the header bellow are not working. I guess this is fine, but wanted to point it just in case we want to do something about.

Same happens for `esp-metadata` and `esp-build`

Reply:
I plan to publish these soon-ish (next few weeks probably), so figured would just get them ready to go. So yes I am aware of this and it was intentional, thank you regardless!